webservices/legal_docs/current_adrs.py

```diff
@@ -0,0 +1,299 @@
+import logging
```

Comment:
1. The module should not be named with a _current_ adjective.
2. Common functions between ADRs and MURs should be refactored into a `utils` module.